pkgs/applications/radio/ax25-apps/default.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, stdenv
+, fetchurl
+, libax25
+, ncurses
+}:
+
+stdenv.mkDerivation rec {
+  pname = "ax25-apps";
+  version = "0.0.8-rc5";
+
+  buildInputs = [ libax25 ncurses ];
+
+  # Due to recent unsolvable administrative domain problems with linux-ax25.org,
```

Comment:
Keeping this here as explained in previous PRs: https://github.com/NixOS/nixpkgs/pull/207585#discussion_r1056980443